Makefile

```diff
@@ -41,6 +44,14 @@ lint: generate
 generate: kustomize yq
 	scripts/generate.sh
 
+.PHONY: deploy
+deploy: generate lint
```

Comment:
nit: `generate` is a dependency of `lint`, so it's redundant.

Reply:
Done.